collection_requirements.rst

```diff
@@ -157,11 +168,22 @@ Branch protections MUST be enforced:
 CI Testing
 ===========
 
-At a minimum ``ansible-test sanity`` MUST be run from the `latest stable ansible-base branch <https://github.com/ansible/ansible/branches/all?query=stable->`_. We suggest to *additionally* run ``ansible-test sanity`` from the ansible/ansible ``devel`` branch so that you find out about new linting requirements earlier.
+* You MUST run ``ansible-test sanity`` from the `latest stable ansible-base/ansible-core branch <https://github.com/ansible/ansible/branches/all?query=stable->`_. 
+* You MUST run CI against all versions of ``ansible-base``/``ansible-core`` that the collection supports.
+* You SHOULD suggest to *additionally* run ``ansible-test sanity`` from the ansible/ansible ``devel`` branch so that you find out about new linting requirements earlier.
+* The sanity tests MUST pass.
+
+  * Adding some entries to the ``test/sanity/ignore*.txt`` file is an allowed method of getting them to pass. 
+  * All entries in ignores.txt MUST have a justification in a comment in the ignore.txt file for each entry.  For example ``plugins/modules/docker_container.py use-argspec-type-path # uses colon-separated paths, can't use type=path``.
+  * Reviewers can block acceptance of a new collection if they don't agree with the ignores.txt entries.
```

Comment:
Hi guys, I couldn't make it at the meeting today. I think _some_ tests for which there should be no excuses: param type not in docs, or list missing elements type. Those are way too simple, and they are the single two major offenders in the ignore files today.

Reply:
yes, I think we agreed in theory to that but decided we could start approve new collections before there was a list of tests which must not be ignored.  If you'd like to propose which tests cannot be ignored, that would be great (use a separate PR and then put it on the meeting agenda).